app/assets/stylesheets/layout_components/index.scss

```diff
@@ -1,13 +1,15 @@
 //Keep this alphabetized
 @import "accordions";
-@import "siteheader";
 @import "banners";
 @import "breadcrumbs";
+@import "call-to-action";
```

Comment:
okay, I just moved them because at the top of the file it said to keep them alphabetical and I noticed they weren't. I'll check it out.